api/pom.xml

```diff
@@ -69,6 +69,10 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-core</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Is this garbage? You only removed classes from the API module, yet you add a dependency?

Reply:
No it s not. It is the dependency to use SimpleBuildItem.
I ll move it to spi module